### DISCUSSION TITLE
rtmp://stream.odysee.com/live don't work

### DISCUSSION THREAD

**Discussion by crewockeez** [datarhei/restreamer#622] [Q&A]
rtmp://stream.odysee.com/live don't work
Key: b5947fba47875a5a7c5f595fce5b7aba673d3325?d=404d6f746f677073706f7476&s=4e838e3499c96cee330dd8a9744200d4795c2aa7d1db2dace9683821f1be0abe627d901305063f477cf57417adb1e1ae6ad1de26f6a516136535e3a14fa4c7c3&t=1695365674

odsee.com

**Comment by svenerbeck** [comment]
Hello,
I added odysee.com to the [feature request list](https://github.com/datarhei/restreamer/discussions/345). We will add an official Publication Service in one of the upcoming updates. 

Cheerio
Sven
